CMU-313/nodebb-f24-aawaa
# [Issues #21,#22,#23] Role Display Feature

User's role appears next to their username when they post a topic and when they post a reply (refresh is required to view tag on replies)
- Issue with updating schema to appropriately have role, leading to test failure.

How to test role displays:
- Run ./nodebb build tpl
- Navigate to “General Discussion”
- Create a new topic to see your role next to your username
- Go into the topic and make a reply
- Refresh the page to view your role next to the comment

Automated testing for role display:
- Test added to test/topics/events.js to ensure correct role assignment to each topic.
- Run test with npm run test

Issues Addressed:
https://github.com/CMU-313/nodebb-f24-aawaa/issues/21
https://github.com/CMU-313/nodebb-f24-aawaa/issues/22
https://github.com/CMU-313/nodebb-f24-aawaa/issues/23

Role shows up next to topic (rendered in categories.tpl):
<img width="553" alt="image" src="https://github.com/user-attachments/assets/7f2af248-b0f2-4d54-a448-4c897fcdbe3a">

Role shows up next to the replies:
<img width="471" alt="image" src="https://github.com/user-attachments/assets/cf23ca82-0e4f-40e2-8b83-6325e2496eae">
